packages/contracts-bedrock/src/L2/CrossL2Inbox.sol

```diff
@@ -55,15 +65,40 @@ contract CrossL2Inbox is ICrossL2Inbox, ISemver, TransientReentrancyAware {
     ///         Equal to bytes32(uint256(keccak256("crossl2inbox.identifier.chainid")) - 1)
     bytes32 internal constant CHAINID_SLOT = 0x6e0446e8b5098b8c8193f964f1b567ec3a2bdaeba33d36acb85c1f1d3f92d313;
 
+    /// @notice The address that represents the system caller responsible for L1 attributes
+    ///         transactions.
+    address internal constant DEPOSITOR_ACCOUNT = 0xDeaDDEaDDeAdDeAdDEAdDEaddeAddEAdDEAd0001;
```

Comment:
Better to use the constant defined in the library: https://github.com/ethereum-optimism/optimism/blob/32eea1b691d66925b028bb72c1564b22a4520d21/packages/contracts-bedrock/src/libraries/Constants.sol#L38

Reply:
yeah i did that initially but they were issues with incompatible Solidity versions

Reply:
Interesting because the `Constants.sol` file is `^0.8.0` so it should be able to be imported here